TWLight/resources/models.py

```diff
@@ -396,7 +395,7 @@ def __str__(self):
     def clean(self):
         if self.agreement_with_terms_of_use and not self.terms_of_use:
             raise ValidationError(
-                "When agreement with terms of use is "
+                "When agreement with terms of use is"
```

Comment:
Removed a space

Reply:
This space is required so that "is" separates from "required".